CHANGELOG.md

```diff
@@ -5,14 +5,29 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+<!-- ---------------------
+      v1.6.0
+     --------------------- -->
+## v1.6.0 - Unreleased
+
+### Added
+
+- Method `saveToFile` becomes `DTensor<T>::saveToFile(std::string pathToFile, Serialisation ser)`, i.e., the user can 
+  choose whether to save the file as a text (ASCII) file, or a binary one
+
+### Changed
+
+ - Method `parseFromFile` renamed to `parseFromFile` (supports text and binary formats)
+
+
 <!-- ---------------------
       v1.5.2
      --------------------- -->
 ## v1.5.2 - 1-12-2024
 
 ### Fixed
 
-- Quick bug bix in `DTensor::parseFromTextFile` (passing storage mode to `vectorFromFile`)
+- Quick bug bix in `DTensor::parseFromFile` (passing storage mode to `vectorFromTextFile`)
```

Comment:
fix

Reply:
This should be `parseFromTextFile` because it's a previous version